funnel/models/venue.py

```diff
@@ -44,6 +44,9 @@ class VenueRoom(BaseScopedNameMixin, db.Model):
     description = MarkdownColumn('description', default=u'', nullable=False)
     bgcolor = db.Column(db.Unicode(6), nullable=False, default=u'229922')
 
+    scheduled_sessions = db.relationship("Session",
+        primaryjoin="and_(Session.venue_room_id == VenueRoom.id, Session.start != None, Session.end != None)", back_populates="venue_room")
```

Comment:
Did you really mean to use `back_populates`? https://docs.sqlalchemy.org/en/latest/orm/relationship_api.html#sqlalchemy.orm.relationship.params.back_populates

Reply:
Let me test couple of things. The complementing property already exists for us.